.pipelines/e2e.yml

```diff
@@ -42,8 +42,13 @@ jobs:
       az account set -s $AZURE_SUBSCRIPTION_ID
 
       set -x
+
+      export PRIVATE_CLUSTER=true
```

Comment:
Are there any e2e tests that need to be adjusted if ingress is made private? If so, should we include the adjustments in this PR as well?

Reply:
Good callout Caden! No tests need to be changed. the only test that can be added is a check for cluster being private. 

Since we have some environments where clusters are public and private in some other environments, Adding this test could have some unexpected repercussions in Fairfax / Local deploy